result_collector/aggregate_results.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import re
+
+def get_number(filename):
+    return re.findall(r'\d+', filename)[0]
+
+def load_previous_cumulative_arrays():
```

Comment:
Again, I'd suggest looking into ROOT IO, especially into TChain as you might deal with many small files on EOS.

Reply:
I would suggest to leave it now, and in case of stable work, we can rewrite that part.

Reply:
Good point. For a first version this should be fine.